sdks/wasm/tests/debugger/debugger-test.cs

```diff
@@ -1,7 +1,7 @@
 using System;
 
 public partial class Math { //Only append content to this class as the test suite depends on line info
-	public static int IntAdd (int a, int b) {
+	public static int IntAdd2(int a, int b) {
```

Comment:
Why was this renamed?

Reply:
`Math.IntAdd` is present in `sample.cs` and `debugger-test.cs`, which means that I can't build `sample.dll` to include all the `debugger-test-*` files. I think we have some documentation referencing `sample.cs`, so instead of removing the method from there, I renamed this one.